medusa/process_tv.py

```diff
@@ -569,7 +572,7 @@ def process_media(processPath, videoFiles, nzbName, process_method, force, is_pr
                                                        u"Continuing the post-process of this file: %s" % cur_video_file)
                         else:
                             associated_files = processor.list_associated_files(cur_video_file_path, subtitles_only=True)
-                            if not [f for f in associated_files if f[-3:] in subtitle_extensions]:
+                            if not [f for f in associated_files if helpers.get_extension(f) in subtitle_extensions]:
```

Comment:
While i like improving the extension checking by using `os.path.splitext`, is it really necessary to use a helper for such a simple function?

Using `os.path.splitext` directly is more explicit and doesn't require an imported helper just to get the index of the result.  

Also to that end, in the `get_extension` helper instead of `[1:]` to strip the `.` from the extension, `.lstrip('.')` is more explicit and easier to understand.